### PR TITLE
Add docs on how to use tracing for local Node dev

### PR DESCRIPTION
**Pull request by AndrewFerr**
- ~Public API changes documented in changelogs (optional)~ N/A

Signed-off-by: Andrew Ferrazzutti <andrewf@element.io>

### CODE REVIEW

**Review by poljar — CHANGES_REQUESTED** [review]
CI seems to complain, could you see what it wants?

File: bindings/matrix-sdk-crypto-nodejs/README.md [hunk 1]
```diff
@@ -184,6 +184,20 @@ See
...
+
+To use tracing in client applications that import these bindings, here's how to do it in
+a local development environment:
+
+-   In this directory, run `npm link` to make your local build of the bindings available to
+    other Node projects on your system
+-   In your client app's source directory, run `npm link @matrix-org/matrix-sdk-crypto-nodejs`
+    to make it use your trace-enabled local build of the bindings
+-   In your client app's source code, add a call to `initTracing` near startup time
+-   Run your app with the `MATRIX_LOG` environment variable set to the desired log level
```
Comment (Hywan) [inline bindings/matrix-sdk-crypto-nodejs/README.md:197]:
```suggestion
- In this directory, run `npm link` to make your local build of the bindings available to
  other Node projects on your system
- In your client app's source directory, run `npm link @matrix-org/matrix-sdk-crypto-nodejs`
  to make it use your trace-enabled local build of the bindings
- In your client app's source code, add a call to `initTracing` near startup time
- Run your app with the `MATRIX_LOG` environment variable set to the desired log level
```

Comment (Hywan) [reply bindings/matrix-sdk-crypto-nodejs/README.md:197]: I don't think the whole `npm link` stuff is important here, is it? Or maybe it is important if `matrix-sdk-crypto-nodejs` is a dependency of another project?

Comment (AndrewFerr) [reply bindings/matrix-sdk-crypto-nodejs/README.md:197]: It's the latter.  Should the section title be changed to make that more clear?

**Review by Hywan — APPROVED** [review]

**Review by poljar — APPROVED** [review]

**Pull request merged**